FireRender.Maya.Src/FireRenderObjects.cpp

```diff
@@ -2464,12 +2464,42 @@ void FireRenderEnvLight::Freshen(bool shouldCalculateHash)
 			context()->iblTransformObject = dagPath.transform();
 			//
 
-			if (!GetPlugValue("display", true) || m.light.IsAmbientLight())
```

Comment:
Why check for "display" flag was removed?

Looks like it would change the logic for override creation